usr.bin/xargs/tests/regress.sh

```diff
@@ -21,4 +21,9 @@ REGRESSION_TEST(`0L', `xargs -0 -L2 echo <${SRCDIR}/regress.0.in')
 REGRESSION_TEST(`0P1', `xargs -0 -P1 echo <${SRCDIR}/regress.0.in')
 REGRESSION_TEST(`quotes', `xargs -n1 echo <${SRCDIR}/regress.quotes.in')
 
+REGRESSION_TEST_FREEFORM(`parallel1', `echo /    /    | xargs -n1 -P2 test -d; [ $? = 0 ]')
+REGRESSION_TEST_FREEFORM(`parallel2', `echo /    /foo | xargs -n1 -P2 test -d; [ $? = 1 ]')
+REGRESSION_TEST_FREEFORM(`parallel3', `echo /foo /    | xargs -n1 -P2 test -d; [ $? = 1 ]')
+REGRESSION_TEST_FREEFORM(`parallel4', `echo /foo /bar | xargs -n1 -P2 test -d; [ $? = 1 ]')
```

Comment:
Names like foo and bar are definitely a non-starter. Instead you should use /var/empty for a directory which does exit and /var/empty/nodir (or whatever else of that sort) which does not.

Reply:
thank you, fixed